vdp/pipeline/v1beta/pipeline.proto

```diff
@@ -2177,3 +2218,15 @@ message ComponentRun {
     (google.api.field_behavior) = OPTIONAL
   ];
 }
+
+// GetPipelineRunsTrendRequest is the request message for GetPipelineRunsTrend.
+message GetPipelineRunsTrendRequest {
```

Comment:
I think the design of the trend API should be more aligned with https://github.com/instill-ai/protobufs/blob/56f60368076924ee05ef1628f4549ea74156ee56/core/mgmt/v1beta/metric.proto#L150